src/AppMenuSearch.cc

```
@@ -380,12 +380,19 @@ static int calculateFuzzyScore(const QString &pattern, const QString &text)
     int consecutive = 0;
     int prevMatchIdx = -1;
 
+    const QString pLower = patternLower.isEmpty() ? pattern.toLower() : patternLower;
```

Clang (14.0.6)

[warning] variable 'pLower' is not initialized

(cppcoreguidelines-init-variables)